noxfile.py

```diff
@@ -225,6 +226,148 @@ def dev_coverage_combine(session: nox.Session) -> None:
     session.run("coverage", "combine", *coverage_files)
 
     session.log("Wrote combined coverage database for all tests to '.coverage'.")
+@nox.session
+def release_test_windows_trace(session: nox.Session) -> None:
+    """Test the SxsTrace side-by-side trace utility with cocotb."""
+
+    release_install(session)
+    session.run(
+        "powershell",
+        "-executionpolicy",
+        "bypass",
+        "-File",
+        "tests\\sxs.ps1",
+        external=True
+    )
+
+
+#
+# Release pipeline.
+#
+# - Build wheels (release builds).
+# - Install cocotb from wheel.
+# - Run tests against cocotb installed from the wheel.
+#
+# The release pipeline does not collect coverage, and does not run doctests.
+#
+
+# Directory containing the distribution artifacts (sdist and bdist).
+dist_dir = "dist"
+
+
+@nox.session
+def release_build(session: nox.Session) -> None:
+    """Build a release (sdist and bdist)."""
+    session.notify("release_build_bdist")
+    session.notify("release_build_sdist")
+
+
+@nox.session
+def release_build_bdist(session: nox.Session) -> None:
+    """Build a binary distribution (wheels) on the current operating system."""
+
+    # cibuildwheel 2.4 is broken until is broken for us until
```

Comment:
```suggestion
    # cibuildwheel 2.4 is broken for us until
```

Reply:
Removed this comment since the latest cibuildwheel release should fix things for us.